### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by AnishMishra1**

### CODE REVIEW

File: constants/queries.js
Comment (sachin691) [inline constants/queries.js:1]: why do we have this reset_token query ?

File: controllers/emailFormController.js
Comment (sachin691) [inline controllers/emailFormController.js:1]: remove extra lines and spaces.

File: controllers/usersController.js
Comment (sachin691) [inline controllers/usersController.js:1]: we have removed verify token function

**Review by sachin691 — DISMISSED** [review]

File: controllers/emailFormController.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const emailSent = async (req, res) => {
+  try {
+    const { name, email, subject, message } = req.body;
+
+    if (email === undefined || name === undefined || subject === undefined || message === undefined) {
+      return res.status(206).json({ success: false, payload: { message: "Please fill all details" } });
+    }
+    
+    const adminEmail = process.env.ADMIN_EMAIL;
+    const emailTemplate = (clientEmail, clientMessage, clientSubject) => `
```
Comment (ruck-45) [inline controllers/emailFormController.js:13]: declare this function in utils and import it here

File: controllers/emailFormController.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                                                              <html>
+                                                              <body>
+                                                                <p>hello Kreative Machine Team<p>
```
Comment (ruck-45) [inline controllers/emailFormController.js:16]: Change : <p>Hello KreativeMachine Team,<p>

File: controllers/emailFormController.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+//Local Dependcies
+const { sendEmail } = require("../utils/sendmail");
+
+const emailSent = async (req, res) => {
+  try {
```
Comment (ruck-45) [inline controllers/emailFormController.js:5]: No need for this try block, remove it

File: controllers/emailFormController.js (hunk 1 above)
Comment (AnishMishra1) [reply controllers/emailFormController.js:13]: solve

File: controllers/emailFormController.js (hunk 2 above)
Comment (AnishMishra1) [reply controllers/emailFormController.js:16]: solve

File: controllers/emailFormController.js (hunk 3 above)
Comment (AnishMishra1) [reply controllers/emailFormController.js:5]: solved

**Review by ruck-45 — APPROVED** [review]

**Pull request merged**